src/proxy-ui-api/src/main/java/org/niis/xroad/restapi/service/AccessRightService.java

```diff
@@ -96,6 +96,30 @@ public AccessRightService(LocalGroupRepository localGroupRepository, GlobalConfF
         this.endpointService = endpointService;
     }
 
+    public List<AccessRightHolderDto> getAccessRightHoldersByClient(ClientId clientId)
```

Comment:
Missing javadocs, I think all public (service) methods should have ones? This one might explain the base endpoint -filtering and what it means for the caller.

Reply:
Added javadoc and also a brief definition of base endpoint. Also added base endpoint definition to EndpointType as it contains isBaseEndpoint-method. Don't know if this is the best place.